native/common/include/jp_javaframe.h

```diff
@@ -348,6 +348,12 @@ class JPJavaFrame
 	jsize GetStringUTFLength(jstring a0);
 
 	jint compareTo(jobject obj, jobject obj2);
+
+	jboolean isPackage(string str);
```

Comment:
Can't we pass a const string reference here?

Reply:
Fixed